.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -0,0 +1,74 @@
+name: Backwards Compatibility Tests k-NN
+on:
+  push:
+    branches:
+      - "*"
+      - "feature/**"
+  pull_request:
+    branches:
+      - "*"
+      - "feature/**"
+
+jobs:
+  Restart-Upgrade-BWCTests-k-NN:
+    strategy:
+      matrix:
+        java: [ 11, 17 ]
+        bwc_version : [ "1.0.0", "1.1.0", "1.2.4", "1.3.1" ]
+        opensearch_version : [ "2.0.0-rc1-SNAPSHOT" ]
```

Comment:
can we define multiple values for opensearch_version?

Reply:
yes, we can like we did for bwc_version